pyproject.toml

```diff
@@ -61,6 +61,7 @@ channels = {version = ">=3.0.5", optional = true}
 astunparse = {version = "^1.6.3", python = "<3.9"}
 libcst = {version = ">=0.4.7", optional = true}
 rich = {version = ">=12.0.0", optional = true}
+pyinstrument = "^4.4.0"
```

Comment:
let's make this optional 😊

If I remember correctly we also need to create a group (like we did for django, fastapi)

Reply:
Didn't know about optional dependencies, will check it out and modify it

Reply:
Added, along with the group